types/node/cluster.d.ts

```diff
@@ -1,10 +1,10 @@
+// Requires `esModuleInterop: true`
```

Comment:
I wish there was a better way to handle this, as not everyone runs with `esModuleInterop: true`. You can merge variables and `namespace` declarations as long as the `namespace` itself doesn't contain any *values*, but it's not pretty:
```ts
const cluster: cluster.Cluster;
namespace cluster {
  interface ClusterSettings { ... }
  interface Address { ... }
  interface Worker extends EventEmitter { ... }
  interface Cluster extends EventEmitter {
    Worker: { // constructor for `Worker` class
      new (options?: EventEmitterOptions): Worker;
      prototype: Worker;
    };
    ...
  }
}
export = cluster;
```

Reply:
I will play around with this, thanks!